contrib/init/sysvinit-redhat/balena-engine

```diff
@@ -84,7 +85,7 @@ start() {
 
 stop() {
 	echo -n $"Stopping $prog: "
-	killproc -p $pidfile -d 300 $prog
+	killproc -p $pidfile -d 300 $exec_basename
```

Comment:
I need to test this, but I think that the argument to `killproc` should be the name of the executable (`balenad`) rather than the name of the service (`balena` before the rename, `balena-engine` after the rename).